src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+    private int numberOfColor = random.nextInt(Color.values().length);
```

Comment:
Only `Random` instance should be a field variable, number itself should be a local method var and regenerated in each call of the `getRandomColor()` method